pkg/oc/cli/admin/migrate/storage/storage.go

```diff
@@ -248,6 +249,26 @@ func (o *MigrateAPIStorageOptions) Complete(f kcmdutil.Factory, c *cobra.Command
 		}
 	}
 
+	clientConfig, err := f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	// We do not have a way to access the REST client that dynamic.NewForConfig uses
+	// Thus we cannot use resource.NewClientWithOptions with our flowcontrol.NewFakeAlwaysRateLimiter
+	// To avoid any possibility of rate limiting, use an absurdly high burst and QPS
+	// We handle throttling via our own limiter
+	clientConfigCopy := rest.CopyConfig(clientConfig)
+	clientConfigCopy.Burst = 1 << 24
+	clientConfigCopy.QPS = 1 << 24
```

Comment:
isn't this one even a float.  What does the shift do there? Really hoping it does the right thing, but just write a big number.

Reply:
untyped numeric const works fine as a float.  But I can use all nines...